ui/pages/create-account/connect-hardware/account-list.js

```diff
@@ -73,7 +73,9 @@ class AccountList extends Component {
       HardwareDeviceNames.lattice,
       HardwareDeviceNames.trezor,
       HardwareDeviceNames.oneKey,
-    ].includes(device.toLowerCase());
+    ]
+      .map((name) => name.toLowerCase())
+      .includes(device.toLowerCase());
```

Comment:
Instead of potentially iterating twice, we could use `.some` maybe?

```suggestion
    ]
      .some((name) => name.toLowerCase() === device.toLowerCase())
```

Reply:
Smart! Thanks a lot 🙂